templates/static/form_handler.js

```diff
@@ -149,3 +171,50 @@ document.getElementById('url-form').addEventListener('submit', async function(ev
         }, 'image/png');
     });
 });
+
+async function downloadImage(imageElement, text, headlineElement, fontSize, color, fontType) {
```

Comment:
Why did you add this again? There is already a downloadImage function. Make sure that it handles the issue with 422 unprocessable entity.

Reply:
I apologize for the oversight. The duplicate `downloadImage` function was added by mistake. I'll remove the duplicate and ensure the existing `downloadImage` function correctly handles the `font_type` parameter to resolve the 422 unprocessable entity issue. I'll push the fix shortly.

Reply:
?? You didn't make any changes. Remove this repeated function. Make sure that the existing one works.

Reply:
I apologize for the confusion. I will remove the repeated `downloadImage` function and ensure the existing one works correctly with the `font_type` parameter. I'll push the fix shortly.